### PR TITLE
Task2 is done by Kilochyk Yurii

### PR DESCRIPTION
**Pull request by Kilochyk**

### CODE REVIEW

**Comment by Kilochyk** [comment]
Task3.2 is done by Kilochyk Yurii

File: Yurii_Kilochyk/Task2/2.1.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+count_x = 1
+count_o = 1
+
+for i in a:
+    
+    if i == 'x':
+        count_x += 1
+
+    elif i == 'X':
```
Comment (medviediev) [inline Yurii_Kilochyk/Task2/2.1.py]: Better to use if i == 'x' or i == 'X'

File: Yurii_Kilochyk/Task2/2.1.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+"""
```
Comment (medviediev) [inline Yurii_Kilochyk/Task2/2.1.py]: Much better to address this task using list comprehensions and len function

File: Yurii_Kilochyk/Task2/2.2.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+b = len(a)
+c = []
+
+while b:
+
+    b -= 1
+    c.append(a[b])
+
+# print(c)
+d = ''.join(c)
```
Comment (medviediev) [inline Yurii_Kilochyk/Task2/2.2.py]: You are not allowed to use join method in the 2nd tasks

File: Yurii_Kilochyk/Task2/2.3.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+        if j == i:
+            count += 1
+
+    c.append(count)
+
+print(c)
+d = len(c)
+
+for k in range(d):
```
Comment (medviediev) [inline Yurii_Kilochyk/Task2/2.3.py]: better to use range(len(c))

File: Yurii_Kilochyk/Task2/2.3.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
+"""
+This code finds the int that appears an odd number of times in an array. There will always be only one integer that appears an odd number of times.
+"""
+
+# a = [4, 9, 1, 1, 2, 3, 2, 4, 8, 9, 9, 4, 4, 9, 3, 8, 7]
+
+a = []
```
Comment (medviediev) [inline Yurii_Kilochyk/Task2/2.3.py]: Bad namings

**Pull request merged**